crypto/ppccap.c

```diff
@@ -212,6 +212,22 @@ size_t OPENSSL_instrument_bus2(unsigned int *out, size_t cnt, size_t max)
 #  include <sys/auxv.h>
 #  define OSSL_IMPLEMENT_GETAUXVAL
 # endif
+# if defined(__FreeBSD__)
```

Comment:
Should this be an "else" from the previous "if"?

Reply:
no in fact wrong level on my part.